kfp/kfp_support_lib/src/kfp_support/workflow_support/utils/workflow_utils.py

```diff
@@ -143,6 +143,39 @@ def __init__(self, host: str = "http://localhost:8080"):
         """
         self.kfp_client = Client(host=host)
 
+    def upload_pipeline(
+        self,
+        pipeline_package_path: str = None,
+        pipeline_name: str = None,
+        description: str = None,
+    ) -> models.api_pipeline.ApiPipeline:
+        """
+        Uploads the pipeline
+        :param pipeline_package_path: Local path to the pipeline package.
+        :param pipeline_name: Optional. Name of the pipeline to be shown in the UI.
+        :param description: Optional. Description of the pipeline to be shown in the UI.
+        :return: Server response object containing pipleine id and other information.
+        """
+        try:
+            p = self.kfp_client.upload_pipeline(pipeline_package_path, pipeline_name, description)
```

Comment:
Do you need to check whether pipeline with the given name already exists? And remove the old one?

Reply:
This functionality is implemented in the https://github.com/IBM/data-prep-kit/blob/dev/kfp/kfp_support_lib/src/kfp_support/workflow_support/utils/pipelines_tests_utils.py#L6 file that already exists in the library.

Reply:
would it be cleaner to move this functionality here? With additional flag to overwrite or fail?

Reply:
Sure, will try to move the implementation. Thanks

Reply:
@blublinsky I changed the function as you suggested. Please review again. Thanks